### DISCUSSION TITLE
Calling function inside a prompt with multiple variables

### DISCUSSION THREAD

**Discussion by mohammedtabish0** [microsoft/semantic-kernel#1095] [Q&A]
Ability to call function from a prompt through multiple variables ? As of now we can only send one variable while calling the function and not multiple I am looking something like below example:- 

TextSkill.Summarize looks like :- 
```
Summarize the topic ${variable1} in language ${variable2}
```

And the calling function looks like :- 
```
Please call the function {{TextSkill.Summarize $variable1 $variable2 }}
```

**Comment by evchaki** [comment]
@mohammedtabish0  try this - https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/KernelSyntaxExamples/Example64_MultiplePromptTemplates.cs
